specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.5/TextAnalytics.json

```diff
@@ -0,0 +1,2931 @@
+            "$ref": ".//examples//SuccessfulLanguagesRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/sentiment": {
+      "post": {
+        "summary": "Sentiment",
+        "description": "The API returns a detailed sentiment analysis for the input text. The analysis is done in multiple levels of granularity, start from the a document level, down to sentence and key terms (targets and assessments).",
+        "operationId": "Sentiment",
+        "consumes": [
+          "application/json",
+          "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ModelVersion"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "$ref": "#/parameters/LoggingOptOut"
+          },
+          {
+            "name": "opinionMining",
+            "in": "query",
+            "description": "(Optional) if set to true, response will contain not only sentiment prediction but also opinion mining (aspect-based sentiment analysis) results.",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/StringIndexType"
+          },
+          {
+            "$ref": "#/parameters/MultiLanguageInput"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call results in a document sentiment prediction, as well as sentiment scores for each sentiment class (Positive, Negative, and Neutral)",
+            "schema": {
+              "$ref": "#/definitions/SentimentResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "500": {
+            "description": "Internal error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Successful Sentiment request": {
+            "$ref": ".//examples//SuccessfulSentimentRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "JobManifest": {
+      "properties": {
+        "tasks": {
+          "description": "The set of tasks to execute on the input documents. Cannot specify the same task more than once.",
+          "properties": {
+            "entityRecognitionTasks": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/EntitiesTask"
+              }
+            },
+            "entityRecognitionPiiTasks": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/c"
```

Comment:
```suggestion
                "$ref": "#/definitions/PiiTask"
```

Reply:
This is the issue I had with the regeneration ^

Reply:
changed